OPM/opm-rule-onboarding.ps1

```diff
@@ -0,0 +1,540 @@
+###########################################################################
+#
+# NAME: OPM Rule Onboarding Utility
+#
+# AUTHOR:  Jeff Rechten
+# AUTHOR:  Assaf Miron
+#
+# COMMENT: 
+# This script will import OPM rules from a CSV file using REST API
+#
+# SUPPORTED CYBERARK VERSIONS:
+# CyberArk PVWA v10.4 and above
+#
+# UTILITY VERSION LIMITATIONS:
+# - No validation checking on PlatformID
+# - No validation checking on existing rules
+# - No validation checking on command restrictions
+# - Capability to add OPM rules only, no update OPM rule capability
+#
+###########################################################################
+[CmdletBinding(DefaultParametersetName="Create")]
+param
+(
+	[Parameter(Mandatory=$true,HelpMessage="Please enter your PVWA address (For example: https://pvwa.mydomain.com/PasswordVault)")]
+	#[ValidateScript({Invoke-WebRequest -UseBasicParsing -DisableKeepAlive -Uri $_ -Method 'Head' -ErrorAction 'stop' -TimeoutSec 30})]
+	[Alias("url")]
+	[String]$PVWAURL,
+	
+	[Parameter(Mandatory=$false,HelpMessage="Enter the Authentication type (Default:CyberArk)")]
+	[ValidateSet("cyberark","ldap","radius")]
+	[String]$AuthType="cyberark",
+	
+	[Parameter(Mandatory=$false,HelpMessage="Enter the RADIUS OTP")]
+	[ValidateScript({$AuthType -eq "radius"})]
+	[String]$OTP,
+	
+	[Parameter(Mandatory=$false,ValueFromPipeline=$false,ValueFromPipelineByPropertyName=$true)]
+	[ValidateScript({Test-Path $_})]
```

Comment:
From other experiences I now know that it worth changing this to include -Leaf
This is an example: [ValidateScript( { Test-Path -Path $_ -PathType Leaf -IsValid})]
I just now added it to the onboarding tool :)

Reply:
thanks. updated